src/tasks/runstart.ts

```diff
@@ -296,7 +307,7 @@ export const RunStartQuest: Quest = {
         get("instant_savePantogram", false),
       do: (): void => {
         Pantogram.makePants(
-          "Mysticality",
+          myPrimestat().toString(),
```

Comment:
We've previously defined `mainStatString`. We should probably export that and use that everywhere we need this string instead of invoking the function calls repeatedly

Reply:
Done